Style/Frontend/index.md

```diff
@@ -0,0 +1,1451 @@
+
+  ```jsx
+  // bad
+  const foo = () => {
+    if (x) {
+      return x;
+    } else {
+      return y;
+    }
+  }
+  
+  // bad
+  const foo = () => {
+    if (x) {
+      return x;
+    } else if (y) {
+      return y;
+    }
+  }
+  
+  // bad
+  const foo = () => {
+    if (x) {
+      return x;
+    } else {
+      if (y) {
+        return y;
+      }
+    }
+  }
+  
+  // good
+  const foo = () => {
+    if (x) {
+      return x;
+    }
+  
+    return y;
+  }
+  
+  // good
+  const foo = () => {
+    if (x) {
+      return x;
+    }
+  
+    if (y) {
+      return y;
+    }
+  }
+  
+  // best
+  const foo = () => {
+    return x ?? y;
+  }
+  ```
+
+## Control Statements
+
+- Don't use selection operators in place of control statements for running functions
+
+  ```jsx
+  // bad
+  !isRunning && startRunning();
+  
+  // good
+  if (!isRunning) {
+    startRunning();
+  }
+  
+  // we do this a lot in JSX, fine to do for literals and variables
+  <View>
+    <Text>{hasProperty && 'Property detected'}</Text>
+  </View>
+  ```
+
+## Components & CSS
+
+- Keep JSX as simple and as optimized as possible. There should be no unnecessary views or unused styles. It is often worth refactoring or at least re-analyzing your JSX and CSS styles before making a PR.
+
+  ```jsx
+  // bad
+  <View style={styles.container}>
+  	<View style={styles.content}>
+    	<Text>This is some content</Text>
+    </View>
+  </View>
+  
+  const styles = StyleService.create({
+  	container: {
+      flex: 1,
+    },
+    content: {
+      alignSelf: 'center',
+  	},
+  });
+  
+  // good
+  <View style={[styles.container, styles.content]}>
+   <Text>This is some content</Text>
+  </View>
+  
+  const styles = StyleService.create({
+  	container: {
+      flex: 1,
+    },
+    content: {
+      alignSelf: 'center',
+  	},
+  });
+  
+  // but do we really need two styles?
+  // sometimes it depends on the component if you choose to merge styles, generally it will be more clear if your styles are as atomic as possible, and if you name them as literally as you can
+  // (if a style just applies flex: 1, call it `flex1`, if a style centers a component, call it `center`, etc)
+  <View style={styles.container}>
+   <Text>This is some content</Text>
+  </View>
+  
+  const styles = StyleService.create({
+  	container: {
+      flex: 1,
+      alignSelf: 'center',
+    },
+  });
+  
+  // you can break out layout styles into their own style to apply as needed
+  <View style={styles.flex1}>
+    <Text style={styles.center}>Header</Text>
+   	<View style={styles.row}>
+      <AppIcon></AppIcon>
+    	<Text>This is some content in a line</Text>
+    </View>
+  </View>
+  
+  const styles = StyleService.create({
+  	container: {
+      flex: 1,
+      alignSelf: 'center',
+    },
+  });
+  ```
+
+- Boolean properties that are false by default can be applied by just including their name
```

Comment:
Really it's just shorthand for `backButton={true}` you don't need a default value to do that.

Reply:
Made this more clear, removed the thing about default value